README.md

```diff
@@ -132,61 +144,46 @@ Make sure a Java Development Kit (JDK) is installed on your operating system if
 you want to use PyJNIus on desktop. OpenJDK is known to work, and the Oracle
 Java JDK should work as well.
 
-On windows, make sure `JAVA_HOME` points to your java installation, so PyJNIus
-can locate the `jvm.dll` file allowing it to start java. This shouldn't be
-necessary on OSX and Linux, but in case PyJNIus fails to find it, setting
+On Windows, make sure `JAVA_HOME` points to your Java installation, so PyJNIus
+can locate the `jvm.dll` file allowing it to start Java. This shouldn't be
+necessary on macOS and Linux, but in case PyJNIus fails to find it, setting
 `JAVA_HOME` should help.
 
-Support
--------
-
-If you need assistance, you can ask for help on our mailing list:
-
-* User Group : https://groups.google.com/group/kivy-users
-* Email      : kivy-users@googlegroups.com
-
-We also have a Discord server:
-
-[https://chat.kivy.org/](https://chat.kivy.org/)
-
-Contributing
-------------
-
-We love pull requests and discussing novel ideas. Check out our
-[contribution guide](http://kivy.org/docs/contribute.html) and
-feel free to improve PyJNIus.
+## License
 
-The following mailing list and IRC channel are used exclusively for
-discussions about developing the Kivy framework and its sister projects:
+PyJNIus is [MIT licensed](LICENSE), actively developed by a great
+community and is supported by many projects managed by the 
+[Kivy Organization](https://www.kivy.org/about.html).
 
-* Dev Group : https://groups.google.com/group/kivy-dev
-* Email     : kivy-dev@googlegroups.com
+## Documentation
 
-License
--------
+[Documentation for this repository](https://kivy.github.io/kivy_pong_demo/)
 
-PyJNIus is released under the terms of the MIT License. Please refer to the
-LICENSE file for more information.
+## Support
 
+Are you having trouble using the Kivy framework, or any of its related projects?
+Is there an error you don’t understand? Are you trying to figure out how to use 
+it? We have volunteers who can help!
 
-## Backers
+The best channels to contact us for support are listed in the latest 
+[Contact Us](https://github.com/kivy/kivy/blob/master/CONTACT.md) document.
 
-Thank you to all our backers! 🙏 [[Become a backer](https://opencollective.com/kivy#backer)]
+## Contributing
 
-<a href="https://opencollective.com/kivy#backers" target="_blank"><img src="https://opencollective.com/kivy/backers.svg?width=890"></a>
+Kivy is a large product used by many thousands of developers for free, but it 
```

Comment:
Since `pyjnius` can be also used without knowing what Kivy is, that could be confusing, maybe is better to specify it?

E.g.

PyJNIus is part of the [Kivy](link) ecosystem, and as all the projects managed by the [Kivy Organization] link, is used by many thousand ....